test/parallel/test-runner-concurrency.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+require('../common');
+const { describe, it } = require('node:test');
+const assert = require('assert');
+
+describe('Concurrency option (boolean) = true ', { concurrency: true }, () => {
```

Comment:
@MoLow I think it'll fail on dual cores as well because of the `cpus().length - 1`. I'll make an update to accommodate dual & single core machines. 
and while I'm at it, add test cases for integer based concurrency as well

Reply:
Updated the tests